src/resources/views/columns/permissions.blade.php

```diff
@@ -0,0 +1,51 @@
+{{-- relationships with pivot table (n-n) --}}
+<td>
+    <?php
+
+    // Gets the attribute to display the permission name
+    $attribute = array_get($column, 'attribute');
```

Comment:
If that were named something less generic than `$attribute` the comment above would be somewhat more obvious; or if `$attribute` really does describe what this is, the above comment isn't needed.